charts/velero/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.4.2
 description: A Helm chart for velero
 name: velero
-version: 2.12.17
+version: 2.12.18
```

Comment:
this looks like a new fetaur I would bump to 2.13.0

Reply:
It's a bug fixing, lots of issues have been reported recently and I think we missed the annotation to `kind: Schedule` at PR #109.

Reply:
This is just a bug fix. I hope it could be merged as soon as possible.

Reply:
also would like to see it merged 👍 🙏

Reply:
@cpanato could you please take a look at this PR, thanks :bow: